src/agent/agent.c

```diff
@@ -12,6 +12,21 @@
 
 #include "agent.h"
 
+struct JobTracker {
+        char *job_object_path;
+        job_tracker_callback callback;
+        void *userdata;
+        free_func_t free_userdata;
+        LIST_FIELDS(JobTracker, tracked_jobs);
+};
+
+static bool
```

Comment:
Extra white space here.

Reply:
This is added by out clang formater, so not sure what I can do about it.